.gitlab.mk

```diff
@@ -1,9 +1,14 @@
 TNT_VER=$(shell cat versions/${OS}_${DIST}_${VER})
```

Comment:
May be add information to the commit message that the commit is a preparation for the merge of dockerfiles? Up to you.